QUICK_START_GUIDE.md

```diff
@@ -50,7 +50,7 @@ working on your laptop or machine and is expected to take approximately XX minut
    ```bash
    python3 -m venv venv
    source venv/bin/activate
-   pip install https://github.com/instructlab/instructlab.git@stable
+   pip install git+https://github.com/instructlab/instructlab.git@stable
```

Comment:
This can now just be “pip install instructlab”

Reply:
Updated ❤️